infra/modules/kubevela-addons/definitions/cardano-wallet-connector.cue

```diff
@@ -0,0 +1,171 @@
+parameter: {
```

Comment:
Should there be a parameter to specify the port, with a sane default?

Reply:
I would let this in the dapps certification side.

IMO this isn't related to the cardano-wallet itself as the same as the `db` volume, which is going to be cutted off from here.
```yaml
properties:
  image: ghcr.io/input-output-hk/plutus-certification:PLT-7784-k8s-sched
  imagePullPolicy: Always
  ports:
    - expose: true
      port: 80
      protocol: TCP
```

Reply:
No, I don't mean the port of the service, I mean the cardano wallet port.

Reply:
done